vector/src/main/java/im/vector/activity/VectorRoomInviteMembersActivity.java

```diff
@@ -701,9 +702,87 @@ public void onClick(View v) {
                         }
                     }
 
-                    // Invite each typed email by creating a direct chat
-                    // Stay in the activity if there is at least one contact selected
-                    inviteNoTchapContactsByEmail(emails, mUserIdsToInvite.isEmpty());
+                    // In order to prepare the lookup3Pids,
+                    // We have to specify the type of media : email or phone number.
+                    // In this case, the media type always is an email.
+                    // That's why we create a new list of the same size as the list of emails,
+                    // in which the media type always is an email.
+                    final List<String> medias = new ArrayList<>();
+
+                    for (String email : emails) {
+                        medias.add(ThreePid.MEDIUM_EMAIL);
+                    }
+
+                    showWaitingView();
+
+                    // Check for each email whether there is an associated account with a Matrix id.
+                    mSession.lookup3Pids(emails, medias, new ApiCallback<List<String>>() {
+                        @Override
+                        public void onSuccess(final List<String> pids) {
+                            Log.e(LOG_TAG, "lookup3Pids success " + pids.size());
+
+                            for (int index = 0; index < emails.size();) {
+                                final String email = emails.get(index);
+                                String mxId = pids.get(index);
+
+                                if (!TextUtils.isEmpty(mxId)) {
+                                    // We check if this email has been already invited
+                                    // Here we consider the pendingInvites because we could have a pending invite related to this Tchap user).
```

Comment:
Please replace the comment with:
// We check here if a discussion already exists for this Tchap user.
// We consider the pendingInvites because we could have a pending invite from this Tchap user.